core/src/main/java/com/linecorp/armeria/common/logging/DefaultRequestLog.java

```diff
@@ -573,6 +573,11 @@ public String requestContentPreview() {
         return requestContentPreview;
     }
 
+    @Override
+    public void requestContentPreview(@Nullable String requestContentPreview) {
+        this.requestContentPreview = requestContentPreview;
```

Comment:
- `requireNonNull`
- Should not allow setting after reaching `REQUEST_CONTENT` availability, because otherwise the preview becomes mutable.

Reply:
`requestContentPreview` can be null if the previewer is disabled.